README.md

```diff
@@ -29,6 +29,7 @@ Checkup implements these storage providers:
 - Amazon S3
 - Local file system
 - GitHub
+- SQL (sqlite3)
```

Comment:
When I hear SQL, I think of a SQL server I have to run. Maybe we should rename this to SQLite storage instead.

Reply:
... and postgresql